source/elements/oneDNN/include/dnnl.hpp

```diff
@@ -541,17 +541,15 @@ struct engine {
 /// @{
 
 /// A container for stream attributes.
-struct stream_attr : public handle<dnnl_stream_attr_t> {
-    using handle::handle;
-
+struct stream_attr {
```

Comment:
why?

Reply:
This header file is not supposed to have any mention of `handle<T>` and, especially, should not have any references to the C API, as this is implementation-specific.

Reply:
oh, my bad, i didn't notice we don't have `handle<T>` in other places. sorry for the noise.